src/Components/Contact.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import './Contact.css'
+import ContactList from './ContactList';
+// import UserProfile from './UserProfile';
+
+
+class Contact extends React.Component {
+    render(){
+        return(
+            <div>
+                <div id="contacts">
+                <h2>Contacts</h2>
+                <div id="contact_List">
+                <ContactList/>
```

Comment:
Since the state on the ContactList is hard coded, it defeats the purpose of having a component. You want to create a re-usable component that thats in props that will populate and fill up the information inside.